constructor/winexe.py

```diff
@@ -75,10 +75,6 @@ def pkg_commands(download_dir, dists, py_version, keep_pkgs, attempt_hardlinks,
     yield "Pop $0"
     yield r'SetDetailsPrint both'
 
-    if not keep_pkgs:
-        yield ''
-        yield r'RMDir "$INSTDIR\pkgs"'
```

Comment:
_Important note for the reviewer_: I have moved this logic into `main.nsi.tmpl` above (see accompanying comment there). However, _this code as written wasn't working_. Because it's not using the `/r` flag, it will not remove the directory since it is not empty.

It takes a non-trivial amount of time to remove the package cache, so fixing this results in a slowdown of installer performance. For this reason, it may be best to _default_ to keeping the package cache on Windows.